api/src/test/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceTest.java

```diff
@@ -292,71 +266,9 @@ public void testSyncComplianceTrainingStatus_Absorb_ResyncCausesNoChanges() thro
         DbAccessModuleName.RT_COMPLIANCE_TRAINING, Timestamp.from(completionTime));
   }
 
-  @Test
-  public void testSyncComplianceTrainingStatus_Moodle_UpdatesVerificationOncePerAccessModule()
```

Comment:
These validations are covered by the `testSyncComplianceTrainingStatus_Moodle` and `testSyncComplianceTrainingStatus_Absorb` tests. I don't think tests specifically for this invariant are valuable.